smartmon.sh

```diff
@@ -155,7 +156,7 @@ OUTPUTAWK
 
 format_output() {
   sort |
-    awk -F'{' "${output_format_awk}"
+    awk -F'{' "${output_format_awk} |sed -e 's/%//g"
```

Comment:
Let's do this with `gsub(/%/, "");`, like we do for dash to underscore in the awk.